pkg/config/config.go

```diff
@@ -15,6 +15,9 @@ const (
 	SshKnownHostsEnv            = "SSH_KNOWN_HOSTS"
 	EksaReplicasReadyTimeoutEnv = "EKSA_REPLICAS_READY_TIMEOUT"
 	ExternalEtcdTimeoutEnv      = "EKSA_EXTERNAL_ETCD_TIMEOUT"
```

Comment:
maybe this has been discussed already, but should this map directly to the AWS CLI env vars like EKSA_REGION would be EKSA_AWS_REGION etc?

Reply:
So the current naming is replacing AWS with EKSA, is your suggestion prefixing EKSA_ in front of the variable names?

Reply:
Yeah I think we should have EKSA_ as prefix. Made the change.